InnerEye/ML/configs/ssl/CXR_SSL_configs.py

```diff
@@ -29,7 +29,7 @@ def __init__(self) -> None:
                          random_seed=1,
                          recovery_checkpoint_save_interval=200,
                          num_epochs=1000,
-                         ssl_training_batch_size=1200,
+                         ssl_training_batch_size=75,
```

Comment:
Over here as well - can we specify the number of GPUs we use for training and effective batch size?